tests/testProjectClass.m

```diff
@@ -553,6 +572,55 @@ function testToStruct(testCase)
             testCase.verifyEqual(projectStruct.contrastNames, contrastNames, 'toStruct method not working');
         end
 
+        function testWriteScript(testCase)
+            % Test that a projectClass object can be written to a script
+            % that can regenerate the object
+
+            % Populates project properties for the tests
+            testCase.populateProject();
+
+            % Verify error for invalid input and write the script
+            testCase.verifyError(@() testCase.project.writeScript(script="invalid.txt"), invalidValue.errorID)
+            testCase.project.writeScript(objName="problem", script="newScript");
+            run("newScript.m");
+    
+            % Test general properties
+            testCase.verifyEqual(testCase.project.experimentName, problem.experimentName, 'experimentName is not correctly regenerated by the script')
```

Comment:
Inconsistent semicolons

Reply:
Which part do you think exactly? As I see it, the tests do not have semicolons but the lines that execute code do, which is the convention we have used elsewhere.

Reply:
I thought we agreed the properties should not have semicolons? Most of the test code seem to have them